### PR TITLE
Add SEO meta tags in reactjs

### LINKED ISSUE #854: SEO Meta Tags for ReactJS
Use something like React Helmet to improve the SEO of the React version of the website.

[Sample Reference](https://www.npmjs.com/package/react-helmet)

### PR DESCRIPTION
**Pull request by stephanieopala**
## Added SEO meta tags in ReactJS
Fixes #854 

## Description
- Import react helmet and added meta tags to the following files;
 app.js, home.js, experience.js, projects.js, research.js, education.js

## Screenshot
Capture

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
1.  Add a title tag
2. If you are importing on different pages modify the title according the page name.
3.  Add a little bit more description in this.
4. Add all meta tags which are present in [this](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/index.html)
5. For guidance refer to this [video](https://www.youtube.com/watch?v=OGoPhwK_P_Q)
6. Try using the parent-child relationship given in the video above

**Comment by stephanieopala** [comment]
> 1. Add a title tag
> 2. If you are importing on different pages modify the title according the page name.
> 3. Add a little bit more description in this.
> 4. Add all meta tags which are present in [this](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/index.html)
> 5. For guidance refer to this [video](https://www.youtube.com/watch?v=OGoPhwK_P_Q)
> 6. Try using the parent-child relationship given in the video above

Noted. Working on the changes.

**Review by anushbhatia — APPROVED** [review]

**Comment by anushbhatia** [comment]
Resolve the conflict

**Comment by stephanieopala** [comment]
> Resolve the conflict

Noted.

File: src/App.js [hunk 1]
```diff
@@ -10,12 +10,33 @@ import Education from './pages/Education/Education'
...
+        <Helmet>
+          <title>John Doe | Home</title>
+          <meta charset="UTF-8"/>
+          <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+          <meta name="keywords" content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
+          <meta name="description" content="This project serves free and beautiful portfolio template for everyone. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+          <meta name="robots" content="index,follow" />
+          <meta name="Googlebot" content="index,follow" />
+          <meta name="AdsBot-Google" content="index,follow" />
+          <meta property="og:site_name" content="https://portfolio.smaranjitghose.codes/"/>
```
Comment (smaranjitghose) [inline src/App.js:29]: https://ahrefs.com/blog/seo-meta-tags/

Use this to improve the overall SEO using the meta tags. Feel free to modify the content

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (stephanieopala) [reply src/App.js:29]: Okay. Working on the changes.

**Comment by smaranjitghose** [comment]
Congratulations on getting your PR merged.

**Pull request merged**